tests/wpt/metadata/performance-timeline/po-disconnect.any.js.ini

```diff
@@ -1,13 +1,13 @@
 [po-disconnect.any.html]
```

Comment:
One last step: since all tests are expected to pass, you need to remove this file. Sorry, I didn't notice this before. Thanks!

Reply:
Alright, I'll do that as soon as I'm home.